packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -99,6 +99,10 @@ describe('Client Test', () => {
     expect(thorClient.getAddress()).toEqual(mainnet_address)
   })
 
+  it('should allow to get the CosmosSDKClient', async () => {
+    expect(thorClient.getCosmosClient()).toBeTruthy()
```

Comment:
@davidp94 Return value of `getCosmosClient` is not a boolean, but an instance of CosmosSDKClient. [toBeInstanceOf](https://jestjs.io/docs/expect#tobeinstanceofclass) might be a better choice to test it (instead of `toBeTruthy`).

Reply:
addressed https://github.com/xchainjs/xchainjs-lib/pull/306/commits/359143b9838f28fb7eb845a5a32c2a51b5ae484f